archive/src/main/java/io/orkes/conductor/dao/postgres/archive/PostgresArchiveDAO.java

```diff
@@ -154,6 +159,38 @@ public void createOrUpdateWorkflow(WorkflowModel workflow) {
         }
     }
 
+    private void updateJsonData(Connection connection, WorkflowModel workflowModel) {
```

Comment:
why is this method here ? why do we need to update completedWithErrors separately ?

Reply:
removed